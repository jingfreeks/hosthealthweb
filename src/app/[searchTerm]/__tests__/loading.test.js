import { render } from "@testing-library/react";
import SearchTermLoading from "../loading";

describe("Search Term Loading Page", () => {
  it("renders Search Term Loading Page to match snapshots", () => {
    const { container } = render(<SearchTermLoading />);
    expect(container).toMatchSnapshot();
  });
});
