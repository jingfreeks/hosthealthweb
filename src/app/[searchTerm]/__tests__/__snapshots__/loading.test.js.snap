// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Term Loading Page renders Search Term Loading Page to match snapshots 1`] = `
<div>
  <main
    class="bg-slate-200 mx-auto max-w-lg p-1 min-h-screen"
  >
    <h2
      class="m-4 text-2xl font-bold"
    >
      Loading...
    </h2>
  </main>
</div>
`;
